apps/web/playwright/caldav-calendar.e2e.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { expect } from "@playwright/test";
+
+import { test } from "./lib/fixtures";
+import { installCaldavCalendar } from "./lib/testUtils";
+
+test.describe.configure({ mode: "parallel" });
+
+test.afterEach(({ users }) => users.deleteAll());
+
+const CALDAV_CALENDER_URL = process.env.E2E_TEST_CALDAV_CALENDER_URL!;
+const CALDAV_CALENDAR_USERNAME = process.env.E2E_TEST_CALDAV_CALENDAR_USERNAME!;
+const CALDAV_CALENDAR_PASSWORD = process.env.E2E_TEST_CALDAV_CALENDAR_PASSWORD!;
```

Comment:
We need to document how one can get all these test credentials and after that we should create these credentials and then merge the PR. cc @zomars @keithwillcode